src/poliastro/tests/tests_twobody/test_sample.py

```diff
@@ -16,17 +17,18 @@ def test_sample_angle_zero_returns_same():
     ss0 = Orbit.from_vectors(Earth, r0, v0)
 
     nu_values = [0] * u.deg
-
     _, rr = ss0.sample(nu_values)
 
-    assert (rr[0].get_xyz() == ss0.r).all()
+    assert_quantity_allclose(rr[0].get_xyz(), ss0.r)
 
 
-@pytest.mark.parametrize("time_of_flight,function", [
-    (1 * u.min, propagate),
-    (40 * u.min, propagate),
+@pytest.mark.parametrize("time_of_flight,method", [
```

Comment:
Prefer stacking `@pytest.mark.parametrize` here to get all the combinations, as explained here: https://docs.pytest.org/en/latest/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions (bottom of the section)